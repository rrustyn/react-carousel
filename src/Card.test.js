import { render } from "@testing-library/react";
import Card from "./Card";

it("makes a card", function () {
  render(<Card caption="pillow" src="img.net" currNum="1" totalNum="1" />);
});

it("matches snapshot", function () {
  const { container } = render(
    <Card caption="pillow" src="img.net" currNum="1" totalNum="1" />
  );
  expect(container).toMatchSnapshot();
});