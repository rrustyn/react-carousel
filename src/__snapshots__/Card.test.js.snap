// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      pillow
    </h4>
    <img
      alt="pillow"
      class="Card-image"
      src="img.net"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      1
      .
    </small>
  </div>
</div>
`;
